Tests/BasicPerformanceTests/SortedArrayPerfTests.swift

```diff
@@ -11,14 +11,44 @@
 import TestSupport
 import Basic
 
+private let magnitude = 5000
+
+private func prng(_ seed: Int) -> Int {
+    return (seed &* 1299827) % magnitude
+}
+
 class SortedArrayPerfTests: XCTestCasePerf {
+
+    private let startSequence = (0..<(magnitude*2)).map(prng)
+    private let overlappingSequence = ((magnitude*1)..<(magnitude*3)).map(prng)
+
+    private var arr: SortedArray<Int> = .init(areInIncreasingOrder: <)
+
+    override func setUp() {
```

Comment:
Avoid using the `setUp` method here, self-contained test cases are nice and there isn't really a lot of duplicated code..